src/React.rei

```diff
@@ -195,7 +195,7 @@ external useSyncExternalStore:
 [@mel.module "react"]
 external useSyncExternalStoreWithServer:
   (
-    ~subscribe: (([@mel.uncurry] (unit => unit)) => ([@mel.uncurry] (unit => unit))),
+    ~subscribe: ([@mel.uncurry] (unit => unit)) => [@mel.uncurry] (unit => unit),
```

Comment:
refmt did this?

Reply:
Yes

Reply:
Tested and looks fine: https://melange.re/unstable/playground/?language=Reason&code=W0BtZWwubW9kdWxlICJyZWFjdCJdCmV4dGVybmFsIHVzZVN5bmNFeHRlcm5hbFN0b3JlOgogICgKICAgIH5zdWJzY3JpYmU6IChbQG1lbC51bmN1cnJ5XSAodW5pdCA9PiB1bml0KSkgPT4gW0BtZWwudW5jdXJyeV0gKHVuaXQgPT4gdW5pdCksCiAgICB%2BZ2V0U25hcHNob3Q6IHVuaXQgPT4gJ3NuYXBzaG90LAogICAgfmdldFNlcnZlclNuYXBzaG90OiBbQG1lbC51bmN1cnJ5XSAodW5pdCA9PiAnc25hcHNob3QpCiAgKSA9PgogICdzbmFwc2hvdCA9CiAgInVzZVN5bmNFeHRlcm5hbFN0b3JlIjsKCmxldCBnZXRTdGF0ZSA9ICgpID0%2BIDIyOwpsZXQgc3Vic2NyaWJlV2l0aE1vY2sgPSBsaXN0ZW5lciA9PiB7CiAgbGV0IF8gPSBsaXN0ZW5lcigpOwogIEpzLmxvZygiY29zaXRhcyIpOwogICgpID0%2BICgpOwp9OwoKbGV0IHNuYXBzaG90ID0KICB1c2VTeW5jRXh0ZXJuYWxTdG9yZSh%2Bc3Vic2NyaWJlPXN1YnNjcmliZVdpdGhNb2NrLCB%2BZ2V0U25hcHNob3Q9Z2V0U3RhdGUpOwo%3D&live=off